mongorestore/oplog.go

```diff
@@ -35,6 +35,15 @@ type oplogContext struct {
 	txnBuffer  *txn.Buffer
 }
 
+// shouldIgnoreNamespace returns true if the given namespace should be ignored during applyOps.
+func shouldIgnoreNamespace(ns string) bool {
+	if strings.HasPrefix(ns, "config.cache.") || ns == "config.system.sessions" {
+		log.Logv(log.Always, "skipping applying the "+ns+" namespace in applyOps")
+		return true
```

Comment:
The jira ticket seems to say that we should be skipping "config.transactions" as well. Was there something outside of the jira comments that caused us to decide to not skip "config.transactions"?

Reply:
I removed the check for the config.transactions namespace here since transactions seem to be skipped anyway in applyOps.